features/gift_exchanges/challenge_giftexchange.feature

```diff
@@ -543,7 +546,8 @@ Feature: Gift Exchange Challenge
       And I should not see the gift exchange dashboard for "Bad Gift Exchange"
       And no one should have an assignment for "Bad Gift Exchange"
       And no one should be signed up for "Bad Gift Exchange"
-    When I am on the collections page
+    When all indexing jobs have been run
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs. (And if you can change the next line to "I go to the collections page" instead of "I am on," that would be great.)